subxt/src/book/usage/transactions.rs

```diff
@@ -66,8 +66,7 @@
 //! There are two main ways to create a compatible signer instance:
 //! 1. The `subxt_signer` crate provides a WASM compatible implementation of [`crate::tx::Signer`]
 //! for chains which require sr25519 or ecdsa signatures (requires the `subxt` feature to be enabled).
-//! 2. Alternately, Subxt can use instances of Substrate's `sp_core::Pair` to sign things by wrapping
-//! them in a new type pattern and implement `crate::tx::Signer`.
+//! 2. Alternately, implement your own [`crate::tx::Signer`] instance using the `polkadot-sdk` signer.
```

Comment:
On line 76 (sorry, can't comment there) I'd change "Let's see how to use each of these approaches:" into something like "Here's an example of the usual approach to signing:"

And then here, we could have:

```suggestion
//! 2. Alternately, implement your own [`crate::tx::Signer`] instance using something like the `polkadot-sdk` signer.
```

Reply:
I changed this is slightly and add something similar at another place